ui/app/src/state/epics/content.ts

```diff
@@ -182,7 +182,7 @@ const content: CrafterCMSEpic[] = [
           // request (via reloadDetailedItem action)
           !state.content.itemsByPath[payload.path] || type === reloadDetailedItem.type
       ),
-      switchMap(([{ payload }, state]) =>
+      mergeMap(([{ payload }, state]) =>
```

Comment:
@rart the issue was that after publishing there was a `reloadDetailedItem` for each published item, and the switchMap was causing a cancelling of the previous reload, so the state wasn't updated properly.